README.md

```diff
@@ -121,16 +137,16 @@ not required):
        "irods_default_resource" : "demoResc"
    }
    ```
-   Note that the *"irods_host"* and *"irods_port"* identify the iRODS server that the DSI will contact during each request.
+   Note that the *"irods_host"* and *"irods_port"* identify the iRODS server that the DSI will contact during each request. Be sure to set the *irods_default_resource*, this variable is not set when you create the file with *iinit* or when you copy it over from another user.
    
 2. As the user who runs the GridFTP server, try an `ils` icommand to verify that 
    the information set in the *irods_environment.json* are fine. If needed, perform 
    an `iinit` to authenticate the iRODS user. 
 
-2. Add the following lines to the GridFTP configuration file ( tipically 
-   *$GLOBUS_LOCATION/etc/gridftp.conf* ):
+3. Add the following lines to the GridFTP configuration file (typically 
+   *$GLOBUS_LOCATION/etc/gridftp.conf*):
    ```
-   $LD_LIBRARY_PATH "$LD_LIBRARY_PATH:/preferred_path/iRODS_DSI"
+   $LD_LIBRARY_PATH "$LD_LIBRARY_PATH:/preferred_path/iRODS_DSI/B2STAGE-GridFTP/"
```

Comment:
export DEST_LIB_DIR="/home/\<user\>/iRODS_DSI"

```sh
~/iRODS_DSI/B2STAGE-GridFTP$ ls
ChangeLog.txt        gridmap_iRODS_callout.conf             Makefile
CMakeCache.txt       images                                 README.md
CMakeFiles           install_manifest.txt                   setup.sh
cmake_install.cmake  libglobus_gridftp_server_iRODS.so      setup.sh.template
CMakeLists.txt       libglobus_gridftp_server_iRODS.so.1.8  testirodsmap
DSI                  libgridmap_iRODS_callout.so            testpidmanager
external             libgridmap_iRODS_callout.so.1.8
```

```sh
~/iRODS_DSI/B2STAGE-GridFTP$ md5sum *so*
ec1d7a5c835e089c1375c37414cc0b77  libglobus_gridftp_server_iRODS.so
ec1d7a5c835e089c1375c37414cc0b77  libglobus_gridftp_server_iRODS.so.1.8
e5aa248d5421e266dc9ccbeb3f0f715b  libgridmap_iRODS_callout.so
e5aa248d5421e266dc9ccbeb3f0f715b  libgridmap_iRODS_callout.so.1.8

```
It is funny, since the ```iRODS_DSI``` contains exactly the same so-files:
```sh
~/iRODS_DSI$ md5sum *so*
ec1d7a5c835e089c1375c37414cc0b77  libglobus_gridftp_server_iRODS.so
ec1d7a5c835e089c1375c37414cc0b77  libglobus_gridftp_server_iRODS.so.1.8
e5aa248d5421e266dc9ccbeb3f0f715b  libgridmap_iRODS_callout.so
e5aa248d5421e266dc9ccbeb3f0f715b  libgridmap_iRODS_callout.so.1.8
```